genny/ci/templates/-dot-circleci/config.yml.tmpl

```diff
@@ -0,0 +1,62 @@
+version: 2
+jobs: 
+  build:
+    docker:
+      - image: circleci/golang:1.12
+      {{ if eq .opts.DBType "postgres" -}}
+      - image: circleci/postgres:9.6-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: {{.opts.App.Name.File}}_test
+      {{- end }}
+
+    environment:
+      TEST_RESULTS: /tmp/test-results
+
+    steps:
+      - checkout
+      - run: mkdir -p $TEST_RESULTS
+
+      - restore_cache:
+          keys:
+            - v1-pkg-cache
+
+      - run: go get github.com/jstemmer/go-junit-report
+
+      {{ if eq .opts.DBType "postgres" -}}
+      - run: go get github.com/lib/pq
```

Comment:
Would these 2 libraries be needed for any other apps ?

Reply:
Just the tests I think? If that's what you're asking?

Reply:
I might be wrong but tipically buffalo migrations use `soda` to run migrations.

Reply:
Let me play around with some options - this works for me - but I'll see if I can find something that is a bit more stock that works.